OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -742,6 +708,7 @@ public BrazeConfig getBrazeConfig() {
     }
 
     @NonNull
+    @SuppressWarnings("unused")
     public EndToEndConfig getEndToEndConfig() {
```

Comment:
as this method is not used we can remove, OR make the parity with iOS.

Reply:
we are using this method in [Credentials.Java](https://github.com/openedx/edx-app-android/blob/master/OpenEdXMobile/src/androidTest/java/org/edx/mobile/test/feature/data/Credentials.java#L27) for testing purpose.